src/lib.js

```diff
@@ -29,10 +29,10 @@ var dePseudify = (function () {
              */
             '::?-(?:moz|ms|webkit|o)-[a-z0-9-]+'
         ],
-        pseudosRegex = new RegExp(ignoredPseudos.join('|'), 'g');
+        pseudosRegex = new RegExp('([^\\\\])(' + ignoredPseudos.join('|') + ')', 'g');
```

Comment:
With the string and RegExp escaping of `\`, it's pretty hard to read this regex at a glance; can you add a comment above this line something like this:

```js
// Actual regex is of the format: /([^\\])(:hover|:focus)/g
```